test/general/TestLocations.py

```diff
@@ -14,3 +15,15 @@ def testCreateDuplicateLocations(self):
             if locations:
                 self.assertLessEqual(locations.most_common(1)[0][1], 1,
                                      f"{world_type.game} has duplicate of location {locations.most_common(1)}")
+
+    def testLocationsInDatapackage(self):
+        """Tests that created locations not filled before fill starts exist in the datapackage."""
+        for game_name, world_type in AutoWorldRegister.world_types.items():
+            if game_name in {"Final Fantasy"}:
+                continue
+            with self.subTest("Game", game_name=game_name):
+                multiworld = setup_default_world(world_type)
+                locations = multiworld.get_unfilled_locations()  # do unfilled locations to avoid Events
+                for location in locations:
+                    self.assertIn(location.name, world_type.location_name_to_id)
+                self.assertLessEqual(len(locations), len(world_type.location_name_to_id))
```

Comment:
```suggestion
```
this line does nothing. if all locations are `in` the datapackage and there are no duplicates, the length is always `<=`

Reply:
fair was mostly adding it for safety